recursio-client/Rendering/PathCSGPolygon.gd

```diff
@@ -0,0 +1,9 @@
+extends CSGPolygon
+
+
+var color setget set_color
+
+
+# Adapter to make `ColorManager.color_object_by_property` integrate neatly
```

Comment:
This wouldn't be necessary, the ColorManager can also deal with methods: `ColorManager.color_object_by_method(color_name, $Path/PathCSGPolygon,material_override,  "set_shader_param", ["color"])` instead of line 25 in GhostPath.gd should work as well. But nothing that has to be changed, just wanted to note it.

Reply:
Ah nice, will keep that in mind, thx!